src/dope-node/blockchain/blockchain.go

```diff
@@ -0,0 +1,28 @@
+package blockchain
+
+import (
+	"fmt"
+	"log"
+)
+
+var blockchain = []Block{}
+
+func AddBlock(content string) {
+	var newBlock Block
+
+	if len(blockchain) == 0 {
+		newBlock = createGenesisBlock(content)
+		log.Println("Created genesis block: " + newBlock.ToString())
+	} else {
+		newBlock = createBlock(blockchain[len(blockchain)-1], content)
+		log.Println("Created new block: " + newBlock.ToString())
+	}
+
+	blockchain = append(blockchain, newBlock)
+}
```

Comment:
Isn't there any possible way that adding block would fail?

Reply:
Don;t worry about it ahaha